exporter/splunkhecexporter/config.go

```diff
@@ -46,6 +47,20 @@ type OtelToHecFields struct {
 	SeverityNumber string `mapstructure:"severity_number"`
 }
 
+// HecHeartbeat defines the heartbeat information for the exporter
+type HecHeartbeat struct {
+	// Interval represents the time interval for the heartbeat interval. If nothing or 0 is set,
+	// heartbeat is not enabled.
+	// A heartbeat is an event sent to _internal index with metadata for the current collector/host.
+	Interval time.Duration `mapstructure:"hec_heartbeat_interval"`
+
+	// ObservabilityName is the prefix will be used to observe the success or failure of the heartbeat
+	ObservabilityName string `mapstructure:"observability_name"`
+
+	// ObservabilityDimensions is the dimensions that will be used for the heartbeat observability
+	ObservabilityDimensions map[string]string `mapstructure:"observability_dimensions"`
```

Comment:
Are there dimensions in hec? Should it be "fields" or maybe "attributes" keeping w/ otel conventions?

Reply:
can change to "fields" instead of "Dimensions".

I have been calling it `Observability` instead of `metrics` to avoid confusion with the pmetrics that are processed by the exporter.